blue_moveit_bringup/launch/left_moveit.launch

```diff
@@ -0,0 +1,40 @@
+<launch>
+  <arg name="version" default="2" doc="1 or 2"/>
+  <!-- By default, we are not in debug mode -->
+  <arg name="debug" default="false" />
+
+  <node
+    name="$(anon blue_controller_spawner)"
+    pkg="blue_controller_manager"
+    type="spawner.py"
+    output="screen"
+    ns="left_arm">
+    <rosparam param="start_controllers">
+      - blue_controllers/joint_trajectory_controller
+      - blue_controllers/gripper_controller
+    </rosparam>
+  </node>
+
+  <!-- Run the main MoveIt executable without trajectory execution (we do not have controllers configured by default) -->
+  <include file="$(find blue_moveit_config)/launch/move_group.launch">
+    <arg name="allow_trajectory_execution" value="true"/>
+    <arg name="fake_execution" value="false"/>
+    <arg name="info" value="true"/>
+    <arg name="debug" value="$(arg debug)"/>
+    <arg name="blue_side" value="blue_left" />
+    <arg name="version" value="$(arg version)" />
+  </include>
+
+  <!-- Run Rviz and load the default config to see the state of the move_group node -->
+  <include file="$(find blue_moveit_config)/launch/moveit_rviz.launch">
```

Comment:
Perhaps the rviz include can be separated/removed as well? It doesn't really seem core to running MoveIt